pr_check.sh

```diff
@@ -17,23 +17,29 @@
 # --------------------------------------------
 # Options that must be configured by app owner
 # --------------------------------------------
-APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
-COMPONENT_NAME="ccx-smart-proxy"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
-IMAGE="quay.io/cloudservices/ccx-smart-proxy"
+export APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="insights-results-smart-proxy"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+export IMAGE="quay.io/cloudservices/ccx-smart-proxy"
 
-IQE_PLUGINS="ccx"
-IQE_MARKER_EXPRESSION="smoke"
-IQE_FILTER_EXPRESSION=""
+export IQE_PLUGINS="ccx"
+export IQE_MARKER_EXPRESSION="smoke" # ccx_data_pipeline_smoke does not exits (at least yet) as marker in the plugin
+export IQE_FILTER_EXPRESSION=""
+export IQE_CJI_TIMEOUT="30m"
 
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
 
-# Temporary stub
-mkdir artifacts
-echo '<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="0" skipped="0" tests="1" time="0.014" timestamp="2021-05-13T07:54:11.934144" hostname="thinkpad-t480s"><testcase classname="test" name="test_stub" time="0.000" /></testsuite></testsuites>' > artifacts/junit-stub.xml
+# Build the image and push to quay
+source $CICD_ROOT/build.sh
 
-# Install bonfire repo/initialize
-# CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
-# curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+# Run the unit tests with an ephemeral db
+# source $APP_ROOT/unit_test.sh
+
+# Deploy rbac to an ephemeral namespace for testing
+source $CICD_ROOT/deploy_ephemeral_env.sh
 
-# source $CICD_ROOT/build.sh
-# source $CICD_ROOT/deploy_ephemeral_env.sh
-# source $CICD_ROOT/smoke_test.sh
+# Run somke tests with ClowdJobInvocation
+# component name needs to be re-export to match ClowdApp name (as bonfire requires for this)
+export COMPONENT_NAME="ccx-smart-proxy" 
+source $CICD_ROOT/cji_smoke_test.sh
```

Comment:
Is there any reason to use 'cji_smoke_test.sh' and not just 'smoke_test.sh'?

Reply:
"smoke_test.sh'" as been [deprecated](https://github.com/RedHatInsights/bonfire/blob/master/cicd/examples/pr_check_template.sh#L46) in favor of "'cji_smoke_test.sh"